packages/react-ui/components/Select/__stories__/Select.stories.tsx

```diff
@@ -151,6 +151,21 @@ Simple.story = {
   },
 };
 
+export const Disabled: CSFStory<JSX.Element> = () => <Select disabled />;
+Disabled.story = {
+  name: 'disabled',
+  parameters: {
+    creevey: {
+      tests: {
+        async disabledSelect() {
+          const element = await this.browser.findElement({ css: '.dropdown-test-container' });
+          await this.expect(await element.takeScreenshot()).to.matchImage();
+        },
+      },
+    },
+  },
+};
+
```

Comment:
Предлагаю немного изменить стори - добавить `value` и более однозначно обозначить `placeholder`.
Также, можно убрать параметры для криви, т.к. дефолтных настроек будет достаточно.

```suggestion
export const Disabled: CSFStory<JSX.Element> = () => (
  <>
    <Select disabled items={['value']} value="value" />
    <Select disabled placeholder="placeholder" />
  </>
);
```

Получится что-то такое:
![image](https://user-images.githubusercontent.com/2708211/113349861-17c01e00-9352-11eb-8c8d-8237223dacc8.png)